eval/http.go

```diff
@@ -265,6 +268,56 @@ func GetAttribute(ctx *hcl.EvalContext, content *hcl.BodyContent, name string) (
 	return seetie.ValueToString(val), nil
 }
 
+func GetBody(ctx *hcl.EvalContext, content *hcl.BodyContent) (string, string, error) {
+	attr, ok := content.Attributes["json_body"]
+	if ok {
+		val, err := attr.Expr.Value(ctx)
+		if err != nil {
+			return "", "", err
+		}
+
+		val, err1 := stdlib.JSONEncodeFunc.Call([]cty.Value{val})
```

Comment:
This encode method seems to have unordered json results. We should find ja way to pass the evaluated result to `json.Marshal` which orders the map keys lexicographically. This applies to all `json_encode` or `json_body` use-cases.

cc: @filex

Reply:
no, the current implementation is ordered lexicographically. I would rather have the "natural" order of the properties as they have been written in HCL or in the incoming JSON string.

we should move that topic to a separate issue.

Reply:
The order is irrelevant in JSON.